### ISSUE TITLE
Is the message "This package is NOT registered on PyPI." necessary?

### ISSUE THREAD

**Issue by khink** [zestsoftware/zest.releaser#286]
On releasing a new package, zest.release says: `INFO: This package is NOT registered on PyPI.`

This made me doubt if i had to register it beforehand, so just to be sure, i did:
```
python setup.py sdist register upload -r pypi
running sdist
running egg_info
...
Server response (410): Project pre-registration is no longer required or supported, upload your files instead.
running upload
...
Server response (200): OK
```

So it seems registration isn't needed. So what's the use of the INFO message?

**Comment by reinout** [comment]
The wording should be fixed. The intention is still good :-)

There's an explicit check when uploading to pypi whether the package is already there. In that case we want an explicit `yes` from you. This hopefully avoids uploading an internal private package by accident. 

The actual "register" step has already been removed for pypi (though you can enable it for older private indexes). But the word "register" is still here in the message.

**Comment by reinout** [comment]
I've released a fix as 6.15.1!
